DEV_SETUP.md

```diff
@@ -490,12 +490,6 @@ Occasionally you will see an app specified with a `#`, like `app_manager#b3`. Th
 http://localhost:8000/mocha/<app_name>/<config>  // (e.g. http://localhost:8000/mocha/app_manager/b3)
 ```
 
-### Continuous JavaScript testing
-
-By running the `watch` command, it's possible to continuously run the JavaScript test suite while developing
-
-    $ grunt watch:<app_name>  // (e.g. grunt watch:app_manager)
```

Comment:
I thought watch was a feature of mocha. Is this not possible anymore?

Reply:
not with the way the current integration works